kitsune/sumo/static/sumo/js/flagit.js

```diff
@@ -1,15 +1,16 @@
 document.addEventListener('DOMContentLoaded', () => {
-    const { reasonFilter, flaggedQueue, topicDropdown, updateStatusButton } = {
+    let isInitialLoad = true;
```

Comment:
When I remove this line and its related code below, this PR works great.

Reply:
I want to see if I can get some help with this money by simply replying to get that from my account details for the use of the 6 different wallets I've been thinking about what I have been in the future for the use of this amount of money for the payment of money for delivery to my bank account details for my self and others who are interested in the bank account details for my self bank of Queensland BSB 124047 acc22513293